packages/components/order-tracking/src/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import component from './order-tracking.component';
+
+import './index.scss';
+
+const moduleName = 'orderTracking';
+const componentName = 'orderTrackingComponent';
+
+angular
+  .module(moduleName, [])
+  .component(componentName, component)
+  .run(/* @ngTranslationsInject:json ./translations */);
+
+export {
+  moduleName as module,
+  componentName as component,
```

Comment:
I don't mind, but I know others have already blamed me for doing this 😅

Reply:
the truth is a cannot remember the reason why i did that Oo it looks not necessary since importing the module could be enough, i'll remove it